wpa_supplicant/ctrl_iface_zephyr.c

```diff
@@ -126,8 +126,22 @@ static void wpa_supplicant_ctrl_iface_receive(int sock, void *eloop_ctx,
 	size_t reply_len = 0;
 
 	buf = os_zalloc(CTRL_IFACE_MAX_LEN + 1);
-	if (!buf)
+	if (!buf) {
+		/* Expect this to fail in stress test cases, so use MSG_DEBUG
+		 * instead of MSG_ERROR.
+		 */
+		wpa_printf(MSG_DEBUG, "Failed to allocate memory for ctrl_iface receive buffer");
+		/* Drain the socket */
+		res = recv(sock, NULL, 0, MSG_TRUNC);
+		if (res < 0) {
+			wpa_printf(MSG_DEBUG, "recvfrom(ctrl_iface): %s",
+					strerror(errno));
+			break;
```

Comment:
`break;` seems to be invalid here, since it will result in the next `recv` being called with a NULL buf again.